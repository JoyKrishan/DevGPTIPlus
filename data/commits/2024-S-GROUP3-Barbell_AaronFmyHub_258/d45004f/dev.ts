// import users from '../mock-data/user.json'
// import workoutLog from '../mock-data/workoutLog.json'
// import workout from '../mock-data/workoutTemplate.json'
    // for (let i = 0; i < workout.length; i++) {
    //   await prisma.workoutTemplate.createMany({
    //     data: workout[i] as Prisma.WorkoutTemplateCreateManyInput[],
    //   })
    // }
    await prisma.workoutTemplate.createMany({
      data: workoutTemplate as Prisma.WorkoutTemplateCreateManyInput[],
    })
    await prisma.workoutLog.createMany({
      data: workoutLog as Prisma.WorkoutLogCreateManyInput[],
    })
    logger('add', 'workoutLog')
