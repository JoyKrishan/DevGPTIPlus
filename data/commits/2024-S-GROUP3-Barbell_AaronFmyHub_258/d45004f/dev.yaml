        version: 1.0.1-snapshot.v498dc07(@types/react@18.2.75)(eslint@8.53.0)(expo-application@5.8.4)(expo-auth-session@5.4.0)(expo-constants@15.4.6)(expo-web-browser@12.8.2)(react-dom@18.2.0)(react-native@0.73.6)(react@18.2.0)(typescript@5.3.3)
        specifier: ^1.21.0
        version: 1.21.0(react-native@0.73.6)
        specifier: ~50.0.17
        version: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
        specifier: ~5.8.4
        version: 5.8.4(expo@50.0.17)
        version: 5.4.0(expo@50.0.17)
        specifier: ~15.4.6
        version: 15.4.6(expo@50.0.17)
        version: 3.3.11(expo@50.0.17)
        version: 11.10.3(expo@50.0.17)
        version: 6.2.2(expo@50.0.17)
        version: 3.4.8(expo-constants@15.4.6)(expo-linking@6.2.2)(expo-modules-autolinking@1.10.3)(expo-status-bar@1.11.1)(expo@50.0.17)(react-dom@18.2.0)(react-native-reanimated@3.6.3)(react-native-safe-area-context@4.8.2)(react-native-screens@3.29.0)(react-native@0.73.6)(react@18.2.0)
        version: 12.8.1(expo@50.0.17)
        version: 0.26.4(expo-modules-autolinking@1.10.3)(expo@50.0.17)
        version: 0.24.12(expo@50.0.17)
  /@clerk/clerk-expo@1.0.1-snapshot.v498dc07(@types/react@18.2.75)(eslint@8.53.0)(expo-application@5.8.4)(expo-auth-session@5.4.0)(expo-constants@15.4.6)(expo-web-browser@12.8.2)(react-dom@18.2.0)(react-native@0.73.6)(react@18.2.0)(typescript@5.3.3):
      expo-application: 5.8.4(expo@50.0.17)
      expo-auth-session: 5.4.0(expo@50.0.17)
      expo-constants: 15.4.6(expo@50.0.17)
      expo-web-browser: 12.8.2(expo@50.0.17)
  /@expo/cli@0.17.10(@react-native/babel-preset@0.73.21)(expo-modules-autolinking@1.10.3):
    resolution: {integrity: sha512-Jw2wY+lsavP9GRqwwLqF/SvB7w2GZ4sWBMcBKTZ8F0lWjwmLGAUt4WYquf20agdmnY/oZUHvWNkrz/t3SflhnA==}
      '@expo/config': 8.5.6
      '@expo/config-plugins': 7.9.1
      '@expo/metro-config': 0.17.7(@react-native/babel-preset@0.73.21)
      '@expo/prebuild-config': 6.8.1(expo-modules-autolinking@1.10.3)
  /@expo/config-plugins@7.9.1:
    resolution: {integrity: sha512-ICt6Jed1J0tPYMQrJ8K5Qusgih2I6pZ2PU4VSvxsN3T4n97L13XpYV1vyq1Uc/HMl3UhOwldipmgpEbCfeDqsQ==}
    dependencies:
      '@expo/config-types': 50.0.0
      '@expo/fingerprint': 0.6.0
      '@expo/json-file': 8.3.0
      '@expo/plist': 0.1.0
      '@expo/sdk-runtime-versions': 1.0.0
      '@react-native/normalize-color': 2.1.0
      chalk: 4.1.2
      debug: 4.3.4(supports-color@8.1.1)
      find-up: 5.0.0
      getenv: 1.0.0
      glob: 7.1.6
      resolve-from: 5.0.0
      semver: 7.6.0
      slash: 3.0.0
      slugify: 1.6.6
      xcode: 3.0.1
      xml2js: 0.6.0
    transitivePeerDependencies:
      - supports-color
    dev: false

  /@expo/config@8.5.6:
    resolution: {integrity: sha512-wF5awSg6MNn1cb1lIgjnhOn5ov2TEUTnkAVCsOl0QqDwcP+YIerteSFwjn9V52UZvg58L+LKxpCuGbw5IHavbg==}
    dependencies:
      '@babel/code-frame': 7.10.4
      '@expo/config-plugins': 7.9.1
      '@expo/config-types': 50.0.0
      '@expo/json-file': 8.3.0
      getenv: 1.0.0
      glob: 7.1.6
      require-from-string: 2.0.2
      resolve-from: 5.0.0
      semver: 7.5.3
      slugify: 1.6.6
      sucrase: 3.34.0
    transitivePeerDependencies:
      - supports-color
    dev: false

  /@expo/metro-config@0.17.7(@react-native/babel-preset@0.73.21):
    resolution: {integrity: sha512-3vAdinAjMeRwdhGWWLX6PziZdAPvnyJ6KVYqnJErHHqH0cA6dgAENT3Vq6PEM1H2HgczKr2d5yG9AMgwy848ow==}
    peerDependencies:
      '@react-native/babel-preset': '*'
    dependencies:
      '@babel/core': 7.24.4
      '@babel/generator': 7.24.4
      '@babel/parser': 7.24.4
      '@babel/types': 7.24.0
      '@expo/config': 8.5.6
      '@expo/env': 0.2.3
      '@expo/json-file': 8.3.0
      '@expo/spawn-async': 1.7.2
      '@react-native/babel-preset': 0.73.21(@babel/core@7.23.9)(@babel/preset-env@7.23.9)
      babel-preset-fbjs: 3.4.0(@babel/core@7.24.4)
      chalk: 4.1.2
      debug: 4.3.4(supports-color@8.1.1)
      find-yarn-workspace-root: 2.0.0
      fs-extra: 9.1.0
      getenv: 1.0.0
      glob: 7.2.3
      jsc-safe-url: 0.2.4
      lightningcss: 1.19.0
      postcss: 8.4.38
      resolve-from: 5.0.0
      sucrase: 3.34.0
    transitivePeerDependencies:
      - supports-color
    dev: false

  /@expo/prebuild-config@6.8.1(expo-modules-autolinking@1.10.3):
    resolution: {integrity: sha512-ptK9e0dcj1eYlAWV+fG+QkuAWcLAT1AmtEbj++tn7ZjEj8+LkXRM73LCOEGaF0Er8i8ZWNnaVsgGW4vjgP5ZsA==}
    peerDependencies:
      expo-modules-autolinking: '>=0.8.1'
    dependencies:
      '@expo/config': 8.5.6
      '@expo/config-plugins': 7.9.1
      '@expo/config-types': 50.0.0
      '@expo/image-utils': 0.4.1
      '@expo/json-file': 8.3.0
      debug: 4.3.4(supports-color@8.1.1)
      expo-modules-autolinking: 1.10.3
      fs-extra: 9.1.0
      resolve-from: 5.0.0
      semver: 7.5.3
      xml2js: 0.6.0
    transitivePeerDependencies:
      - encoding
      - supports-color
    dev: false

  /@react-native-async-storage/async-storage@1.21.0(react-native@0.73.6):
    resolution: {integrity: sha512-JL0w36KuFHFCvnbOXRekqVAUplmOyT/OuCQkogo6X98MtpSaJOKEAeZnYO8JB0U/RIEixZaGI5px73YbRm/oag==}
  /babel-preset-expo@10.0.2(@babel/core@7.23.9):
    resolution: {integrity: sha512-hg06qdSTK7MjKmFXSiq6cFoIbI3n3uT8a3NI2EZoISWhu+tedCj4DQduwi+3adFuRuYvAwECI0IYn/5iGh5zWQ==}
  /expo-application@5.8.4(expo@50.0.17):
    resolution: {integrity: sha512-mH0gSlFzMuiT20TkZnBppzUtdF6TFHwcaTRnvkSZanN9iTOeEEfQQ60hHT4lBizHjFWRs/QYZVtXGvkQ/cyMpg==}
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-asset@9.0.2(expo@50.0.17):
      expo-constants: 15.4.6(expo@50.0.17)
      expo-file-system: 16.0.9(expo@50.0.17)
  /expo-auth-session@5.4.0(expo@50.0.17):
      expo-application: 5.8.4(expo@50.0.17)
      expo-constants: 15.4.6(expo@50.0.17)
      expo-crypto: 12.8.1(expo@50.0.17)
      expo-linking: 6.2.2(expo@50.0.17)
      expo-web-browser: 12.8.2(expo@50.0.17)
  /expo-constants@15.4.6(expo@50.0.17):
    resolution: {integrity: sha512-vizE69dww2Vl0PTWWvDmK0Jo2/J+WzdcMZlA05YEnEYofQuhKxTVsiuipf79mSOmFavt4UQYC1UnzptzKyfmiQ==}
      '@expo/config': 8.5.6
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-crypto@12.8.1(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-dev-client@3.3.11(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
      expo-dev-launcher: 3.6.9(expo@50.0.17)
      expo-dev-menu: 4.5.8(expo@50.0.17)
      expo-dev-menu-interface: 1.7.2(expo@50.0.17)
      expo-manifests: 0.13.2(expo@50.0.17)
      expo-updates-interface: 0.15.3(expo@50.0.17)
  /expo-dev-launcher@3.6.9(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
      expo-dev-menu: 4.5.8(expo@50.0.17)
      expo-manifests: 0.13.2(expo@50.0.17)
  /expo-dev-menu-interface@1.7.2(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-dev-menu@4.5.8(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
      expo-dev-menu-interface: 1.7.2(expo@50.0.17)
  /expo-file-system@16.0.9(expo@50.0.17):
    resolution: {integrity: sha512-3gRPvKVv7/Y7AdD9eHMIdfg5YbUn2zbwKofjsloTI5sEC57SLUFJtbLvUCz9Pk63DaSQ7WIE1JM0EASyvuPbuw==}
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-font@11.10.3(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-keep-awake@12.8.2(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-linking@6.2.2(expo@50.0.17):
      expo-constants: 15.4.6(expo@50.0.17)
  /expo-manifests@0.13.2(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
      '@expo/config': 8.5.6
  /expo-router@3.4.8(expo-constants@15.4.6)(expo-linking@6.2.2)(expo-modules-autolinking@1.10.3)(expo-status-bar@1.11.1)(expo@50.0.17)(react-dom@18.2.0)(react-native-reanimated@3.6.3)(react-native-safe-area-context@4.8.2)(react-native-screens@3.29.0)(react-native@0.73.6)(react@18.2.0):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
      expo-constants: 15.4.6(expo@50.0.17)
      expo-linking: 6.2.2(expo@50.0.17)
      expo-splash-screen: 0.26.4(expo-modules-autolinking@1.10.3)(expo@50.0.17)
  /expo-secure-store@12.8.1(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-splash-screen@0.26.4(expo-modules-autolinking@1.10.3)(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-updates-interface@0.15.3(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo-updates@0.24.12(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
      expo-manifests: 0.13.2(expo@50.0.17)
      expo-updates-interface: 0.15.3(expo@50.0.17)
  /expo-web-browser@12.8.2(expo@50.0.17):
      expo: 50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21)
  /expo@50.0.17(@babel/core@7.23.9)(@react-native/babel-preset@0.73.21):
    resolution: {integrity: sha512-eD8Nh10BgVwecU7EVyogx7X314ajxVpJdFwkXhi341AD61S2WPX31NMHW82XGXas6dbDjdbgtaOMo5H/vylB7Q==}
      '@expo/cli': 0.17.10(@react-native/babel-preset@0.73.21)(expo-modules-autolinking@1.10.3)
      '@expo/config': 8.5.6
      '@expo/config-plugins': 7.9.1
      '@expo/metro-config': 0.17.7(@react-native/babel-preset@0.73.21)
      babel-preset-expo: 10.0.2(@babel/core@7.23.9)
      expo-asset: 9.0.2(expo@50.0.17)
      expo-file-system: 16.0.9(expo@50.0.17)
      expo-font: 11.10.3(expo@50.0.17)
      expo-keep-awake: 12.8.2(expo@50.0.17)