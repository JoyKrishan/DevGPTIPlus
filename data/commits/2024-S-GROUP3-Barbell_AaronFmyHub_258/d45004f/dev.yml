      - name: Setup Prisma Schema, Migrate, and Seed
          pnpm db:prisma-ci && pnpm db:seed

      - name: Upload api coverage reports to Codecov
        uses: codecov/codecov-action@v4.0.1
        with:
          token: ${{ secrets.CODECOV_TOKEN }}
          slug: UNLV-CS472-672/2024-S-GROUP3-Barbell
          directory: ./packages/api/vitest-coverage/
          fail_ci_if_error: true