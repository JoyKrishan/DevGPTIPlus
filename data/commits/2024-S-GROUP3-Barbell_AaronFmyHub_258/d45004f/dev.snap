        <View>
          <Text>
            Child content goes here
          </Text>
        </View>